tests/test_visitors/test_tokenize/test_comments/test_comment_in_formatted_string.py

```diff
@@ -0,0 +1,121 @@
+from typing import Final
+
+import pytest
+
+from wemake_python_styleguide.compat.constants import PY312
+from wemake_python_styleguide.violations.best_practices import (
+    CommentInFormattedStringViolation,
+)
+from wemake_python_styleguide.visitors.tokenize.comments import (
+    CommentInFormattedStringVisitor,
+)
+
+if not PY312:  # pragma: >=3.12 no cover
+    pytest.skip(
+        reason='comments in fstring was added in 3.12', allow_module_level=True
+    )  # pragma: no cover
+
+quotes: Final[frozenset[str]] = frozenset(("'", '"'))
```

Comment:
```suggestion
quotes = frozenset(("'", '"'))
```

we don't use typing in tests in this project.